### DISCUSSION TITLE
how do I insert and get the inserted record using REST API?

### DISCUSSION THREAD

**Discussion by saloni137** [supabase/supabase#7788] [Questions] [answered]
I am insterting a record and I want the inserted record's id in return to make another API call.
How do I do it?
I tried select=* sending as a parameter but not getting any response

**Comment by GaryAustin1** [comment, marked as answer]
If you go to your API tab in the dashboard, select Bash tab, then a table, you can see you need a Prefer header to return the inserted info:

![image](https://user-images.githubusercontent.com/54564956/179753742-a6df00bf-b38d-48b0-bd4f-a295ff830606.png)

**Reply by saloni137** [reply]
That worked!! thanks
